source/features/show-whitespace.tsx

```diff
@@ -62,7 +62,13 @@ const viewportObserver = new IntersectionObserver(changes => {
 });
 
 function init(): void {
-	for (const line of select.all('.blob-code-inner:not(.rgh-observing-whitespace)')) {
+	const selectors = [
+		'.blob-code-inner', // Code lines
+		'.highlight pre', // Highlighted code blocks in comments
+		'.snippet-clipboard-content pre', // Not highlighted code blocks in comments
+	].join(',');
+
+	for (const line of select.all(`:is(${selectors}):not(.rgh-observing-whitespace, .blob-code-hunk)`)) {
```

Comment:
Are you saying that PR isn't affected? Let's try 

```suggestion
	for (demo) {
```

To check, you'll have to include a `.prepend(1)` to the function that parses the spaces. If the suggestion has two 1s, then it's affected.

Reply:
Interesting, it does seem to work correctly:

<table>
<tr>
	<td><img width="256" alt="Screen Shot 8" src="https://user-images.githubusercontent.com/1402241/135822523-2a135397-9cae-4cd0-8070-2fb16ea10a85.png">

</table>

Reply:
Here's my conclusion:

- `.blob-code-inner` selects lines in _files_ (that is, file content, review suggestions, etc.)
- `.highlight > pre` selects highlighted code blocks in _comments_ (yes, it should have a `>`, since the file content also has `.highlight`, but luckily it is caught by `.rgh-observing-whitespace`).
- `.snippet-clipboard-content > pre` selects _not_ highlighted code blocks in _comments_ (`>` is optional here but I feel it is safer.)